### ISSUE TITLE
[clang-format] SeparateDefinitionBlocks: Always inserts empty lines before 'requires'

### ISSUE THREAD

**Issue by michaelvlach** [llvm/llvm-project#54666]

**Comment by michaelvlach** [comment]
The following:
```
export template<typename T>
requires(!Stringifiable<T> && Iterable<T>) auto operator<<(std::ostream &stream, const T &container) -> std::ostream &
```
will become with LLVM-14:
```
export template<typename T>

requires(!Stringifiable<T> && Iterable<T>) auto operator<<(std::ostream &stream, const T &container) -> std::ostream &
{
```
if `SeparateDefinitionBlocks` is set to `Always`. Attaching clang-format with this setting commented out for reference.

[clang-format.zip](https://github.com/llvm/llvm-project/files/8387294/clang-format.zip)

**Comment by rymiel** [comment]
Fixed by 9aab0db13fb6d21d1b70247a9b5e4cf916ee1c3a
